tests/rptest/clients/kubectl.py

```diff
@@ -129,9 +133,22 @@ def _install(self):
                 'gcp': [
                     'gcloud', 'container', 'clusters', 'get-credentials',
                     f'redpanda-{self._cluster_id}', '--region', self._region
+                ],
+                'azure': [
+                    'bash', '-c',
+                    ("'if ! command -v az &> /dev/null; then "
+                     "echo \"Azure CLI not found. Installing Azure CLI.\" >&2; "
```

Comment:
This feels awkward. Shouldn't we install the Azure CLI in the same place we install `awscli2`/`gcloud`?

Reply:
I agree, I have mentioned it in our docs that this should be improved.. Let me dig into it. To do it properly we should update the template or do it like AWS and GCP. Thank you

Reply:
Actually, we might need to track this part in a separate PR. Here we just check if it is available and install if needed. But we could also update templates to have this pre-installed.

Would you agree?

Reply:
As long as we know that there's still cleanup to be done, I'm good! I just wanted to make sure to note it.